WordPress/Classes/ViewRelated/Comments/CommentTableHeaderView.swift

```diff
@@ -0,0 +1,156 @@
+        hostingController.view.backgroundColor = .clear
+        contentView.addSubview(hostingController.view)
+        contentView.pinSubviewToAllEdges(hostingController.view)
+    }
+
+    enum Constants {
+        static let postCommentSubText = NSLocalizedString(
+            "comment.header.subText.post",
+            value: "Comment on",
+            comment: """
+                Provides a hint that the current screen displays a comment on a post.
+                The title of the post will be displayed below this text.
+                Example: Comment on \n My First Post
+                """
+        )
+
+        static let replyCommentSubTextFormat = NSLocalizedString(
+            "comment.header.subText.reply",
+            value: "Reply to %1$@",
+            comment: """
+                Provides a hint that the current screen displays a reply to a comment.
+                %1$@ is a placeholder for the comment author's name that's been replied to.
+                Example: Reply to Pamela Nguyen
+                """
+        )
+
+        static let commentThreadSubText = NSLocalizedString(
+            "comment.header.subText.commentThread",
+            value: "Comments on",
+            comment: """
+                Sentence fragment.
+                The full phrase is 'Comments on' followed by the title of the post on a separate line.
+                """
+        )
+    }
+}
+
+// MARK: - SwiftUI
+
+private struct CommentHeaderView: View {
+
+    @State var title = String()
+    @State var subtitle: CommentTableHeaderView.Subtitle = .post
+    @State var showsDisclosureIndicator = true
+
+    var body: some View {
+        if #available(iOS 15.0, *) {
+            // Material ShapeStyles are only available from iOS 15.0.
+            content.background(.ultraThinMaterial)
+        } else {
+            ZStack {
+                VisualEffectView(effect: UIBlurEffect(style: .systemUltraThinMaterial))
+                content
+            }
+        }
+    }
+
+    var content: some View {
+        HStack {
+            text
+            Spacer()
+            if showsDisclosureIndicator {
+                disclosureIndicator
+            }
+        }
+        .padding(EdgeInsets(top: 10, leading: 16, bottom: 10, trailing: 16))
+    }
+
+    var text: some View {
+        VStack(alignment: .leading) {
+            Text(subtitle.stringValue)
+                .lineLimit(1)
+                .font(.footnote)
+                .foregroundColor(Color(.secondaryLabel))
+            Text(title)
+                .lineLimit(1)
+                .font(.subheadline)
+                .foregroundColor(Color(.text))
+        }
+    }
+
+    var disclosureIndicator: some View {
+        Image(systemName: "chevron.forward")
+            .renderingMode(.template)
+            .foregroundColor(Color(.secondaryLabel))
+            .font(.caption.weight(.semibold))
+            .imageScale(.large)
+            .padding(.zero)
```

Comment:
Just wondering, is the `.padding(.zero)` here because `Image` has its own default padding?

Reply:
Just tested both, and I can verify that setting the padding to zero does nothing 😆. Likely from when I was tweaking the component to position it correctly. I've removed this line in https://github.com/wordpress-mobile/WordPress-iOS/pull/20753/commits/df561e20b18068867eaf58a9c7627d84421d5ed6. Thanks!